examples/laptop_pipeline.yml

```diff
@@ -104,8 +104,6 @@ inputs:
     photometry_catalog: data/example/inputs/photometry_catalog.hdf5
     photoz_source_training: submodules/RAIL/tests/data/test_dc2_training_9816.hdf5
     photoz_source_testing: submodules/RAIL/tests/data/test_dc2_validation_9816.hdf5
-    photoz_lens_training: submodules/RAIL/tests/data/test_dc2_training_9816.hdf5
```

Comment:
Removing this since the source and lens catalogs are the same in this example so we don't need to train both.